src/Command/GoAccess/AnalyzeCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace FourViewture\PshLogAnalyzer\Command\GoAccess;
+
+use FourViewture\PshLogAnalyzer\Util\PlatformShEnvSelector;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AnalyzeCommand extends Command
+{
+    protected static $defaultName = 'goaccess:analyze';
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Create a GoAccess report')
+            ->addOption(
+                'project',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'The platform.sh project id',
+                null
+            )
+            ->addOption(
+                'lines',
+                'l',
+                InputOption::VALUE_OPTIONAL,
+                'The Number of lines to analyze',
```

Comment:
One excess comma. 

```suggestion
                'The Number of lines to analyze'
```

Reply:
Nope the default is missing - I’m wat further locally, please keep that PR open. Then it’s easier for me to update it.